README.md

```diff
@@ -1,146 +1,272 @@
-# Fondant
+<p align="center">
+    <img src="docs/art/fondant_banner.svg" height="250px"/>
+</p>
+<p align="center">
+    <i>Sweet data-centric foundation model fine-tuning</i>
+    <br>
+    <a href="https://fondant.readthedocs.io/en/latest/"><strong>Explore the docs »</strong></a>
+    <br>
+    <br>
+    <a href="https://discord.gg/HnTdWhydGp"><img alt="Discord" src="https://img.shields.io/discord/1108377020821405707?label=discord&style=flat-square"></a>
+    <a href="https://pypi.org/project/fondant/"><img alt="PyPI version" src="https://img.shields.io/pypi/v/fondant?color=brightgreen&style=flat-square"></a>
+    <a href="https://fondant.readthedocs.io/en/latest/license/"><img alt="License" src="https://img.shields.io/github/license/ml6team/fondant?style=flat-square&color=brightgreen"></a>
+    <a href="https://github.com/ml6team/fondant/actions/workflows/pipeline.yaml"><img alt="GitHub Workflow Status" src="https://img.shields.io/github/actions/workflow/status/ml6team/fondant/pipeline.yaml?style=flat-square"></a>
+    <a href="https://coveralls.io/github/ml6team/fondant?branch=main"><img alt="Coveralls" src="https://img.shields.io/coverallsCoverage/github/ml6team/fondant?style=flat-square"></a>
+</p>
 
-Fondant is a data-centric framework to fine-tune [Foundation Models](https://fsi.stanford.edu/publication/opportunities-and-risks-foundation-models) such as:
+---
+**Fondant helps you create high quality datasets to fine-tune foundation models such as:**
 
-- Stable Diffusion
-- CLIP
-- Large Language Models (LLMs like GPT-3)
-- Segment Anything (SAM)
-- etc.
+- :art: Stable Diffusion  
+- :page_facing_up: GPT-like Large Language Models (LLMs)  
+- :mag_right: CLIP  
+- :scissors: Segment Anything (SAM)  
+- :heavy_plus_sign: And many more
 
-Fondant focuses on data preparation to fine-tune these models.
+## :mouse_trap: Why Fondant?
 
-## Installation
+Foundation models simplify inference by solving multiple tasks across modalities with a simple
+prompt-based interface. But what they've gained in the front, they've lost in the back. 
+**These models require enormous amounts of data, moving complexity towards data preparation**, and 
+leaving few parties able to train their own models.
 
-Fondant can be installed from source using pip:
+We believe that **innovation is a group effort**, requiring collaboration. While the community has 
+been building and sharing models, everyone is still building their data preparation from scratch.
+**Fondant is the platform where we meet to build and share data preparation workflows.**
 
-```
-pip install git+https://github.com/ml6team/fondant.git
-```
+Fondant offers a framework to build **composable data preparation pipelines, with reusable 
+components, optimized to handle massive datasets**. Stop building from scratch, and start 
+leveraging reusable components instead to:
```

Comment:
Proposal to shorten sentence to:
"Stop building from scratch, and start reusing components to:"